cmd/rook/secret.go

```diff
@@ -113,3 +132,91 @@ func getSecret(cmd *cobra.Command, args []string) {
 		rook.TerminateFatal(errors.Wrapf(err, "failed to write secret %q file to %q", secretName, secretPath))
 	}
 }
+
+// cliRotateSecret is the Cobra CLI call
+func cliRotateSecret() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rotate-key [kms-secret-key] [data-device] [metadata-device] [wal-device]",
+		Short: "Rotate a secret from a given KMS",
```

Comment:
I thought we weren't rotating from a KMS yet?

Reply:
> I thought we weren't rotating from a KMS yet?

modified wording,